region/lowernorfair/east.json

```diff
@@ -1796,10 +1887,42 @@
                   "heatProof",
                   "canHeatRun"
                 ]},
-                {"cost": [
-                  {"heatFrames": 200}
-                ]},
-                "Morph"
+                {"cost": [{"heatFrames": 200}]},
+                "Morph",
+                {"or": [
+                  {"and": [
+                    {"cost": [{"heatFrames": 50}]},
```

Comment:
I'm not sure what the implications of this are. Does it mean you are expected to handle the Dessgeega (or take a hit) when arriving at 4?

If so, I think the link from 2 to 4 should be 2 Dessgeega hits when not killing them.

Reply:
Yes and you're right it should be two hits at 2 to 4.